cli/pazuzu/actions/config.go

```diff
@@ -0,0 +1,61 @@
+package actions
+
+import (
+	"errors"
+	"fmt"
+	"github.com/urfave/cli"
+	"github.com/zalando-incubator/pazuzu"
+	"github.com/zalando-incubator/pazuzu/config"
+	"os"
+	"text/tabwriter"
+)
+
+func ConfigShow(c *cli.Context) error {
+	if c.NArg() != 0 {
+		return errors.New("Wrong number of arguments")
```

Comment:
Maybe it makes sense to introduce few custom self-describing error types? 
Just to separate error representation, something like:

```go
if c.NArg() != 0 {
    return WrongArgumentsNumber()
}
```